docs/learning/tutorials/develop-desktop-viewer.md

```diff
@@ -2,19 +2,15 @@
 
 ### Setup
 - [Install necessary prerequisites]($docs/getting-started/development-prerequisites)
-- [Clone imodeljs-samples repo](https://github.com/imodeljs/imodeljs-samples)
-- Have access to a cloud hosted iModel. If you do not have access to one, follow one of our tutorials to [create an online iModel]($docs/learning/tutorials/index.md)
+- [Clone desktop-starter repo](https://github.com/imodeljs/desktop-starter)
 
 ### Build
 - Open the cloned repo in VS Code
-- `cd interactive-app`
-- `cd simple-viewer-app`
-- Edit interactive-app\simple-viewer-app\\.env.local
-    > `imjs_test_imodel` = The name of your iModel<br/>
-
 - `npm install`
-- `npm run build`
 
 ### Run
-- `npm run electron`
+- `npm run start`
+
+The app will open a pre-packaged offline snapshot iModel. You will likely want to use your own snapshot iModel. Follow the [Create a snapshot iModel]($docs/learning/tutorials/create-test-imodel-offline) tutorial. Then open your briefcase by clicking the Home button and selecting "Open Offline Snapshot".
 
+Desktop apps can also open cloud hosted iModels. If you do not have access to one, follow one of our tutorials to [create an online iModel]($docs/learning/tutorials/index.md). Then open your online iModel by selecting "Open Online iModel" and logging in.

```

Comment:
Can we call this a Briefcase everywhere here instead of "online" or "cloud hosted" iModel?  I think at the end of this we may want to provide a link to the documentation on the updated details of Snapshot vs. Briefcase.

Reply:
I feel Offline/Online nomenclature is proper for this level of documentation. It is getting started, someone getting started wouldn't know Snapshot/Briefcase, but everyone knows what Offline/Online would be.

Reply:
True, my concern is that in the Desktop workflow Briefcases can be used both online and offline (a disconnected scenario) and what distinguishes the iModel is whether it's connected to iModelHub or it's a Snapshot.

@kckst8 I know we went over some of this when discussing the now iModel Selection UI, do you have the wording we were looking to use there?